app/src/main/res/layout/fragment_flare_pointer_offset.xml

```diff
@@ -11,10 +11,11 @@
     android:layout_marginBottom="@dimen/bpkSpacingLg"
     app:flarePointerPosition="start">
 
-    <androidx.appcompat.widget.AppCompatImageView
+    <ImageView
```

Comment:
android automatically switches to the `AppCompatImageView` - using `ImageView` makes the content description lint check work